### PR TITLE
[Fix] Improve verbiage used in snackbar notifications for content operations

### LINKED ISSUE #6413: [UI] Improving the notifications in the stackbar.
#### Current Behavior

image
2> Don't capitalize every second letter as it gives a bad feel. Update the notifications as neccessary in the UI. 

#### Desired Behavior

1> Remove the exclamation marks '!' in the stackbar. 
One such place is when the user clones his design.

2> Don't capitalize every second letter as it gives a bad feel. Update the notifications as neccessary in the UI. 

#### Mockups

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by rakshitgondwal**
Signed-off-by: rakshitgondwal <rakshitgondwal3@gmail.com>

**Description**

- Removed the exclamation marks '!' in the stackbar.
- Removed the capitalization of every other word after the first word.

This PR fixes #6413

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Chadha93 — COMMENTED** [review]
@rakshitgondwal Nice work, will you also ensure that this is updated in applications and filters as well?

File: ui/components/MesheryPatterns.js [hunk 1]
```diff
@@ -485,7 +485,7 @@ function MesheryPatterns({
         body : pattern_file,
       }, () => {
         updateProgress({ showProgress : false });
-        enqueueSnackbar("Pattern Successfully Deployed!", {
+        enqueueSnackbar("Pattern successfully deployed", {
```
Comment (Chadha93) [inline ui/components/MesheryPatterns.js:488]: Update Pattern to Design

Comment (rakshitgondwal) [reply ui/components/MesheryPatterns.js:488]: Should I update Pattern to Design on every change or on this particular comment?

**Review by leecalcote — CHANGES_REQUESTED** [review]
@rakshitgondwal and @Chadha93, please ensure that we are not sending a "Success" message type when only a blue "Info" message type should be sent instead. @Chadha93, you'll have to provide instruction here.

@Chadha93 also, please ensure that we:
1. only capitalize the first word in the notification.
2. incorporate the name of the item (design, filter, app) into the notification and enclose in double quotation marks.
3. use the term "design", not the term "pattern".

**Comment by leecalcote** [comment]
Thanks for jumping into the project, @rakshitgondwal 👏

**Comment by Chadha93** [comment]
@rakshitgondwal Let me know if you are not able to make the requested changes by today. I'll take the PR forward.

**Comment by Chadha93** [comment]
> > @rakshitgondwal Let me know if you are not able to make the requested changes by today. I'll take the PR forward.
> 
> Hey @Chadha93, I have changed the term Pattern to Design in the latest commit, for the other things, I guess the capitalization of the first word is also done, but I'm not being able to get around the other things as mentioned by @leecalcote.

No worries @rakshitgondwal I've pushed the desired changes which cover all the points. I highly recommend reviewing it.

**Comment by Chadha93** [comment]
@leecalcote I've incorporated all these points:
1. only capitalize the first word in the notification. :heavy_check_mark: 

2. incorporate the name of the item (design, filter, app) into the notification and enclose in double quotation marks.:heavy_check_mark: 

3. use the term "design", not the term "pattern". :heavy_check_mark: 

Video attached: 

https://user-images.githubusercontent.com/65453826/199335630-f81e2320-d3bf-400a-b983-54e0cbfabb1a.mp4

**Comment by leecalcote** [comment]
Great job, @rakshitgondwal. FYI there is a conversation on this topic in slack under the #meshery-ui channel

**Comment by leecalcote** [comment]
Oh, yeah… and MeshMate @Chadha93, I guess that you did OK, too.

**Comment by leecalcote** [comment]
Oh, yeah… and MeshMate @Chadha93, I guess that you did OK, too.

**Comment by rakshitgondwal** [comment]
> > > @rakshitgondwal Let me know if you are not able to make the requested changes by today. I'll take the PR forward.
> > 
> > 
> > Hey @Chadha93, I have changed the term Pattern to Design in the latest commit, for the other things, I guess the capitalization of the first word is also done, but I'm not being able to get around the other things as mentioned by @leecalcote.
> 
> No worries @rakshitgondwal I've pushed the desired changes which cover all the points. I highly recommend you to review it.

I just reviewed the changes, now I understand what I was missing. Thanks a lot for helping out.

**Comment by leecalcote** [comment]
> > Great job, @rakshitgondwal. FYI there is a conversation on this topic in slack under the #meshery-ui channel
> 
> Thanks a lot, @leecalcote. This was my first contribution to meshery, and I look forward to contributing more in the future. I'll make sure to check the slack channel too.

Awesome. I'm looking forward to you digging your heels in, too. What a great contribution to kick-off your engagement in the project. 👍

**Review by leecalcote — APPROVED** [review]

**Comment by rakshitgondwal** [comment]
For sure @leecalcote

**Comment by Chadha93** [comment]
Yay @rakshitgondwal. :balloon:

**Comment by rakshitgondwal** [comment]
> Yay @rakshitgondwal. 🎈

Thank you for the merge.!

**Pull request merged**